manifests/p/PaprikaApp/Paprika/3/3.2.2/PaprikaApp.Paprika.3.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PaprikaApp.Paprika.3
+PackageVersion: 3.2.2
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://www.paprikaapp.com/downloads/windows/latest/PaprikaSetup.msi

```

```suggestion
  InstallerUrl: https://www.paprikaapp.com/downloads/windows/3.2.2/PaprikaSetup.msi
```
sometimes wingetbot may not be able to update this package so having a versioned link here might be good.